charts/opentelemetry-demo-datadogv2/Chart.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: v2
 name: opentelemetry-demo-datadog
 description: OpenTelemetry Demo Helm Chart for Datadog
 type: application
-version: 0.1.0
+version: 0.1.1
 appVersion: "0.1.0"
 dependencies:
   - name: datadog
     condition: ingest
     version: 3.3.0
     repository: https://helm.datadoghq.com
   - name: opentelemetry-demo
-    version: 0.12.6
+    version: 0.15.4
```

Comment:
`opentelemetry-demo` is on version 0.22.3, do we want to upgrade? https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-demo/Chart.yaml